groovy.properties

```diff
@@ -2,15 +2,22 @@ category=Languages
 description=Enables scanning of Groovy source files
 homepageUrl=https://github.com/Inform-Software/sonar-groovy
 issueTrackerUrl=https://github.com/Inform-Software/sonar-groovy/issues
-publicVersions=1.6
+publicVersions=1.7
 privateVersions=
-archivedVersions=
+archivedVersions=1.6
 
 defaults.mavenGroupId=org.sonarsource.sonar-groovy-plugin
 defaults.mavenArtifactId=sonar-groovy-plugin
 
 1.6.description=Support for modern SonarQube versions, dependency updates
-1.6.sqVersions=[7.9,LATEST]
+1.6.sqVersions=[7.9,8.9.*]
```

Comment:
This should end with `LATEST`... unless you really mean to have this version marked incompatible when 9.0 comes out?